libmozdata/config.py

```diff
@@ -32,7 +41,9 @@ def get_default_paths(self):
 
     def get(self, section, option, default=None, type=str):
         if not self.config.has_option(section, option):
-            return default
+            if default is not None:
+                return default
+            return super().get(section, option)
```

Comment:
Yuo can remove the if above

```suggestion
            return default or super().get(section, option)
```